### PR TITLE
RSPEC TEST

### PR DESCRIPTION
**Pull request by Artbsc1992**
# Achievements

- Create validations for model files.
- Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: spec/models/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    end
+  end
+
+  context 'save and retrive valid comment' do
+    it 'save valid comment and retirive it from database' do
+      comment.save
+      expect(Comment.find(comment.id)).to be_an_instance_of(Comment)
+    end
+  end
+end
```
Comment (ahmetbozaci) [inline spec/models/comment_spec.rb]: - [ ] Kindly add a test for the comment counter method. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods).

File: spec/models/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  end
+
+  context 'most recent comment method works as expected' do
+    post = Post.first
+    it 'it returns a valid type' do
+      recent_comment = Post.most_recent_comments(post)
+      expect(recent_comment.length).to be_a_kind_of(Integer)
+    end
+  end
+end
```
Comment (ahmetbozaci) [inline spec/models/post_spec.rb]: - [ ] Kindly add a test for post counter method.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Artbsc1992  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You added validations to models. ✔️
- You created tests and all tests are passed. ✔️

image

___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Artbsc1992 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**